File: Advanced Methods of Programming/Client-ServerCSharp/SharedDLL/Domain/Angajat.cs
using System;

namespace SharedDLL.Domain
{
    [Serializable]
    public class Angajat : IHasID<string>
    {
        private string UserAngajat;
        private string PasswordAngajat;

        public Angajat(string User, string Password)
        {
            this.UserAngajat = User;
            this.PasswordAngajat = Password;
        }

        public string GetId() { return this.UserAngajat; }

        public string GetPasswordAngajat() { return this.PasswordAngajat; }

        public override string ToString() { return this.UserAngajat + " " + this.PasswordAngajat; }
    }
}

File: Advanced Methods of Programming/JavaClientServer/Model/src/main/java/Model/Angajat.java
package Model;

import java.io.Serializable;
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name = "angajat")
public class Angajat implements HasId<String>, Serializable {
    @Id
    @Column(name = "userAngajat")
    private  String userAngajat;

    @Column(name = "passwordAngajat")
    private  String passwordAngajat;

    public Angajat() {}

    public Angajat(String userAngajat, String passwordAngajat) {
        this.userAngajat = userAngajat;
        this.passwordAngajat = passwordAngajat;
    }

    public String getId() {
        return userAngajat;
    }

    public void setId(String userAngajat) {
        this.userAngajat = userAngajat;
    }

    public String getPasswordAngajat() {
        return passwordAngajat;
    }

    public void setPasswordAngajat(String passwordAngajat) {
        this.passwordAngajat = passwordAngajat;
    }

    @Override
    public String toString() {
        return userAngajat+" "+passwordAngajat;
    }
}
